package.json

```diff
@@ -59,16 +62,13 @@
     "markdown-it-task-lists": "2.1.1",
     "prettier": "2.2.1",
     "rimraf": "3.0.2",
-    "supertest": "6.1.3",
+    "supertest": "^6.1.3",
```

Comment:
Those pinned dependencies have been enforced by renovate bot.
So it is fine while probably renovate will propose again to fix it to a certain release.